### ISSUE TITLE
Remove script preview from Inspector

### ISSUE THREAD

**Issue by gnysek** [YoYoGames/GameMaker-Feature-Requests#1470]
### Is your feature request related to a problem?

Inspector is a very useful thing, and will be even more useful with new code editor, but there one feature which is completely unnecessary.

It's "Script (read-only)" field. It shows what's already visible on screen second time:

![obraz](https://github.com/YoYoGames/GameMaker-Feature-Requests/assets/524094/16a02b40-c311-4528-9fc1-c08dc5e7d27d)

There's no need in such thing, and also this casues syntax highlighting and all that stuff to be displayed which may leads to bugs/performance issues <sup>[[1]](https://forum.gamemaker.io/index.php?threads/2023-1100-beta-release-thread.106652/#post-645156)</sup>.

### Describe the solution you'd like

Solution A)

- just remove it

Solution B)

- list functions that are inside script

Solution C)
- list functions, globals, constructors methods and properties of structs in script

**Comment by Totobal5** [comment]
Yeah i think the same, is innecesary functionality and sometimes it get in the way of inspector functionality opening scripts when i am using inspectors for objects or viceversa. 

Please remove

**Comment by yoshikins** [comment]
I would genuinely love to see solution B/C, but iirc that ability or at least something similar to it is coming with new code editor, so getting rid of the script inspector may be the best solution because there really isn't any point to it

**Comment by gnysek** [comment]
If new code editor have such features, then I will strike those two proposals as soon as it would enter open beta.

**Comment by yoshikins** [comment]
Re: So based off of the preview yyg had in their 2023 GameMaker Update video, the script inspector itself isn't getting any changes to it but the new code editor with have a drop down tab containing what is essentially your solution C. I thought I should specify
